sdk/identity/identity-brokered-auth/samples/v1-beta/javascript/index.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates usage of @azure/identity-brokered-auth and @azure/identity
+ * packages for WAM MSA support.
+ */
+
+const { InteractiveBrowserCredential, useIdentityPlugin } = require("@azure/identity");
+const { nativeBrokerPlugin } = require("@azure/identity-brokered-auth");
+const dotenv = require("dotenv");
+
+// Load the plugin
+useIdentityPlugin(nativeBrokerPlugin);
```

Comment:
Can we scope this to credentials that are using it v/s other credentials that aren't using this?

Reply:
done